documentacion/Especificación API Arteflujo OpenAPI 3.0.md

```
@@ -0,0 +1,183 @@
+  \- url: https://api.arteflujo.com/v1  
+    description: Servidor de producción
+
+components:  
+  securitySchemes:  
+    bearerAuth:  
+      type: http  
+      scheme: bearer  
+      bearerFormat: JWT
+
+  schemas:  
+    EstadoAcceso:  
```

markdownlint-cli2 (0.21.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)